components/groups.js

```diff
@@ -125,6 +125,62 @@ SteamCommunity.prototype.leaveGroup = function(gid, callback) {
 	});
 };
 
+SteamCommunity.prototype.getAllAnnouncements = function(gid, time, callback) {
```

Comment:
This needs to have "group" in the method name to make it apparent where it's getting announcements from. For example, `getAllGroupAnnouncements`.